src/commands/logstream/enableFileLogging.ts

```diff
@@ -25,7 +24,7 @@ export async function enableFileLogging(context: IEnableFileLoggingContext, node
         node = <SiteTreeItem>node.parent;
     }
 
-    const siteNode: SiteTreeItem | TrialAppTreeItem = node;
+    const siteNode: SiteTreeItem = node;
```

Comment:
nit: This and fullName are kind of pointless `consts` now

Reply:
`siteNode` still helps
<img width="612" alt="Screen Shot 2021-02-02 at 1 30 37 PM" src="https://user-images.githubusercontent.com/11282622/106665216-df22e480-655a-11eb-966d-6f82014be7aa.png">